hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHAWithData.java

```diff
@@ -104,6 +105,30 @@ public void testMultipartUpload() throws Exception {
     testMultipartUploadWithOneOmNodeDown();
   }
 
+  @Test
+  public void testOMHAMetrics() throws InterruptedException {
+    Thread.sleep(2000);
```

Comment:
Having a sleep in a test can be a cause of 'flaky' tests depending on the load of system at time of test.  Can we change this to be a sleep conditional and use a `genericTestUtils.waitFor(() -> {...}, checkEveryMillis, waitForMillis)`?

Reply:
Thanks for pointing that out. I fixed it!